mobile/Components/MapMarker.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { StyleSheet, Animated, View } from 'react-native';
+import { Marker } from 'react-native-maps';
+
+const styles = StyleSheet.create({
+  markerWrap: {
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  marker: {
+    width: 20,
+    height: 20,
+    borderRadius: 5,
+    backgroundColor: 'rgba(130,4,150, 0.9)',
+  },
+});
+
+const MapMarker = ({ state, interpolations }) => {
```

Comment:
The preferred syntax is writing components/screens with`function name()` and all of its helper functions in the arrow function format `const name = () => `